datafusion/core/src/config.rs

```diff
@@ -307,7 +315,8 @@ impl ConfigOptions {
     pub fn get_string(&self, key: &str) -> String {
         match self.get(key) {
             Some(ScalarValue::Utf8(Some(s))) => s,
-            _ => "".into(),
+            Some(s) => s.to_string(),
+            _ => "".to_string(),
```

Comment:
If the `key` is not set, should we return an Error?

Reply:
I think a more common thing in Rust would be to return `Option<String>` so if the string wasn't set the caller can react to that.